Src/PerformanceCollector/Shared/Implementation/QuickPulse/Service contract/MetricPoint.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ManagementServices.RealTimeDataProcessing.QuickPulseService
+{
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    internal struct MetricPoint
```

Comment:
Consider using MetricTelemetry instead. It will ensure consistence of how you collect telemetry